postman/collection-dir/cybersource/Flow Testcases/Happy Cases/Scenario13 Incremental auth/.meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "childrenOrder": [
+    "Payments-Create Copy",
+    "Incremental Authorization Copy",
+    "Payments - Retrieve Copy 2"
```

Comment:
Can we name it better by not using "Copy"? You also would have to rename the folders as well.